code/controllers/admin.dm

```diff
@@ -59,11 +59,14 @@ ADMIN_VERB(debug_controller, R_DEBUG, "Debug Controller", "Debug the various per
 	var/list/controllers = list()
 	var/list/controller_choices = list()
 
-	for (var/datum/controller/controller in world)
+	for (var/var_key in global.vars)
+		var/datum/controller/controller = global.vars[var_key]
+		if(!istype(controller))
+			continue
 		if (istype(controller, /datum/controller/subsystem))
```

Comment:
Pulling in a suggestion made over on [the monke version of this PR](https://github.com/Monkestation/Monkestation2.0/pull/5204).
```suggestion
		if(!istype(controller) || istype(controller, /datum/controller/subsystem))
```